pyproject.toml

```diff
@@ -53,7 +53,7 @@ docs = [
     "jupyter_sphinx==0.5.3",
     "numpydoc==1.8.0",
     "matplotlib==3.10.3",
-    "ansys-sphinx-theme==1.5.2",
+    "ansys-sphinx-theme>=1.0.0,<1.5",
```

Comment:
@ansys/pyansys-core 

We can not use ansys-sphinx-theme version 1.5.2 as search box has disappeared from [PyFluent docs](https://fluent.docs.pyansys.com/). 

We have copied this condition from [PyAEDT repo](https://github.com/ansys/pyaedt/blob/main/pyproject.toml#L81).

Reply:
@Revathyvenugopal162 Done. Thank you.